tools/exporters/clinical_export.py

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python
+
+"""Export MedBook clinical data into rectangular files.  all fields from the form will be exported
+   Export metadata files for cbio import
+
+Usage:
+./clinical_export.py --sample_group_id (sample_group_id) --form_id (form_id)  --work-dir (directory)
```

Comment:
there is no rule that says the clinical data and expression data have to have the same number of samples. You could have more samples in the clinical file that do not exist in the expression data. Good practice to define a sample group for the data freeze.